addons/escoria-ink-bridge/api/escoria_ink_bridge.gd

```diff
@@ -33,7 +33,7 @@ func is_story_valid(ink_file: String) -> bool:
 	return not _get_story(ink_file) == null
 
 
-func run_story(ink_file: String):
+func run_story(ink_file: String, knot: String):
```

Comment:
Add a default value (, knot := "") for knot here to make it backwards compatible.